docs/en/01-how-to-build/cross_build_aarch64.md

```diff
@@ -0,0 +1,108 @@
+# Ubuntu Cross Build aarch64
+
+mmdeploy chose ncnn as the inference backend for aarch64 embedded linux devices. There are two parts:
+
+Host
+
+- model conversion
+- cross build SDK and demo for embedded devices
+
+Device
+
+- Run converted model
+
+## 1. Model Convert on Host
+
+Refer to the doc to install [mmdeploy](../01-how-to-build/) and [mmcls](https://github.com/open-mmlab/mmclassification), and convert resnet18 for model package
+
+```bash
+$ export MODEL_CONFIG=/path/to/mmclassification/configs/resnet/resnet18_8xb32_in1k.py
+$ export MODEL_PATH=https://download.openmmlab.com/mmclassification/v0/resnet/resnet18_8xb32_in1k_20210831-fbbb1da6.pth
+
+# Convert resnet18
+$ cd /path/to/mmdeploy
+$ python tools/deploy.py \
+  configs/mmcls/classification_ncnn_static.py \
+  $MODEL_CONFIG \
+  $MODEL_PATH \
+  tests/data/tiger.jpeg \
+  --work-dir resnet18 \
+  --device cpu \
+  --dump-info
+```
+
+## 2. Cross Build on Host
+
+It is recommended to compile directly with the script
+
+```bash
+$ sh -x tools/scripts/ubuntu_cross_build_aarch64.sh
+```
+
+The following is the manual process corresponding to the script:
+
+a) Install aarch64 build tools
+
+```bash
+$ sudo apt install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+```
+
+b) Cross build opencv and install to /tmp/ocv-aarch64
+
+```bash
+$ git clone https://github.com/opencv/opencv --depth=1 --branch=4.x --recursive
```

Comment:
I suggest using a fixed version of opencv as we did for other third-party libraries

Reply:
done.